scripts/verify-test-resource.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+# Set environment variable for GitHub action to recognise colours
+export TERM=xterm
+
+# Colours for stdout
+RED=$(tput setaf 1)
+GREEN=$(tput setaf 2)
+WHITE=$(tput setaf 7)
+NC=$(tput sgr0)
+RED_BG=$(tput setab 1)
+
+# Path to test resources
+TEST_RESOURCES_PATH=src/test/resources
+
+# Temporary directory for saving compiled classes
+TEMP_DIR_COMPILED_RESOURCES=temp
+mkdir -p $TEMP_DIR_COMPILED_RESOURCES
```

Comment:
Dangerous, what if someone runs this locally and there's already a directory called `temp`? Since you don't check if the directory was actually created, you could later end up deleting a preexisting directory.

Use `mktemp -d` instead, it creates a temporary directory in `/tmp` and echos the path to it.

```bash
TEMP_DIR_COMPILED_RESOURCES=$(mktemp -d)
```

Reply:
Better solution :)